### DISCUSSION TITLE
Does Patchman support Ubuntu 24.04 please?

### DISCUSSION THREAD

**Discussion by OpenThink1** [furlongm/patchman#601] [General]
Does Patchman support Ubuntu 24.04 yet please?

**Comment by furlongm** [comment]
Not yet... I'll start a branch soon with the changes required.

**Reply by MrAhlefeld** [reply]
Hi furlongm,

Looking forward to the Ubuntu 24.04 branch  :) keep up the good work!

Any idea when you will start the branch ?

**Reply by furlongm** [reply]
`main` has all the changes in it. I need to create and upload packages, then the `p4-install` branch will get merged with updated instructions.
